en/identity-server/7.2.0/docs/apis/index.md

```diff
@@ -34,6 +34,10 @@ This is a sample cURL command template for the request.
 curl -X GET "https://localhost:9443/api/server/v1/applications?limit=30&offset=0" -H "accept: application/json" -H "Authorization: Basic <Base64(username:password)>"
 ```
 
+!!! warning "Security advisory: Avoid using super admin credentials for API authentication"
```

Comment:
Why not just say: `Avoid using super admin credentials for API authentication`, without the `Security advisory:` part. Delivers the same message while being consistent with other places of the doc IMO.

Reply:
+1 fixed with c07d83ab063b6dbee306c06b72f984d1b1059be8